composer.lock

```diff
@@ -1735,16 +1735,16 @@
         },
         {
             "name": "simplesamlphp/saml2",
-            "version": "v3.3.8",
+            "version": "v3.4.2",
```

Comment:
Consider not updating SAML2 for this release. Or identify this does not harm any current behaviour

Reply:
Reverted it and only updated xmlseclibs